tools/generate_docs_netlify.sh

```diff
@@ -7,10 +7,13 @@ cd "$(dirname "$0")/../"
 
 brew install pandoc
 
-pip install qdrant-client
+curl -sSL https://install.python-poetry.org | python3 -
+export PATH="/opt/buildhome/.local/bin:$PATH"
+poetry install
 
 pip install sphinx==4.5.0
+pip install ipython
```

Comment:
why isn't it in pyproject.toml dev group?

Reply:
This is needed only for the docs build. Nothing else. Should we include that in the dev group? If yes, will add this, Sphinx, pygments and other docs dependencies there.

Reply:
they are already there

Reply:
Will add these to a new `docs` group in a different PR